cluster_config/configurations.yaml.tpl

```diff
@@ -33,6 +33,15 @@ Configurations:
   Properties:
     "hbase.emr.storageMode": "s3"
     "hbase.emr.readreplica.enabled": "true"
+- Classification: "emrfs-site"
+  Properties:
+    "fs.s3.consistent": "true"
+    "fs.s3.consistent.metadata.read.capacity": "800"
+    "fs.s3.consistent.metadata.write.capacity": "200"
+    "fs.s3.maxConnections": "10000"
+    "fs.s3.consistent.retryPolicyType": "exponential"
```

Comment:
You don't want to use exponential backoff. We've seen in testing that it can lead to wait times of half an hour. Use fixed instead.

Reply:
Changed to fixed , Thanks Connor